source/blog/2017-05-03-minishift-intro.html.md

```diff
@@ -0,0 +1,129 @@
+
+It is actually a single binary which uses [libmachine](https://github.com/docker/machine/tree/master/libmachine) to provision a VM with a local, single node, [OpenShift Origin](https://github.com/openshift/origin) cluster. It uses [OpenShift local cluster management](https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md) underneath to bring up single node cluster.
+
+It's a platform agnostic CLI tool that supports multiple hypervisors like KVM for Linux, Xhyve for Mac OS, HyperV for Windows and VirtualBox in all OS platforms. Detailed instructions about it can be found in the [Minishift documentation](https://github.com/Minishift/minishift#prerequisites).
+
+## Installing Minishift
+
+1. Install your preferred VM platform on your workstation.
+
+    **Note**: You need to install additional docker-machine driver for 'kvm' or 'xhyve'. See [Docker Machine drivers](https://docs.openshift.org/latest/minishift/getting-started/docker-machine-drivers.html) guide to install it properly.
+
+1. Download the archive for your OS from the Minishift [releases page](https://github.com/minishift/minishift/releases/tag/v1.0.0) and unpack it.
+1. Copy the contents of the directory to your preferred location.
+1. Add the minishift binary to your _PATH_ environment variable.
+
+To install in specific OS, refer to the [minishift installation guide](https://docs.openshift.org/latest/minishift/getting-started/installing.html).
+
+## Quickstart with Minishift
+
+1. Get started as follows:
+
+    ```
+    $ minishift start
+     Starting local OpenShift cluster using 'kvm' hypervisor...
+     ...
+        OpenShift server started.
+        The server is accessible via web console at:
+            https://192.168.99.128:8443
+
+        You are logged in as:
+            User:     developer
+            Password: developer
+
+        To login as administrator:
+            oc login -u system:admin
+    ```
+
+    This will:
+    - Download the latest [_minishift-b2d-iso_](https://github.com/Minishift/minishift-b2d-iso) (~40 MB)
+    - Starts a VM using libmachine
+    - Download OpenShift client binary (`oc`)
+    - Cache both `oc` and ISO into your `$HOME/.minishift/cache` folder
+    - Finally provisions OpenShift single node cluster in your workstation
+
+    **Note:** This step also logs you in as `developer` in the `myproject` default project.
+
+1. Use `minishift oc-env` to display the command you need to type into your shell in order to add the oc binary to your _PATH_.
+   The output of `oc-env` will differ depending on OS and shell type.
+
+    ```
+    $ minishift oc-env
+    export PATH="/Users/john/.minishift/cache/oc/v1.5.0:$PATH"
+    # Run this command to configure your shell:
+    # eval $(minishift oc-env)
+    ```
+
+1. Run `eval $(minishift oc-env)` as per above instructions to setup your shell.
+    ```
+    $ eval $(minishift oc-env)    # Might differ based on OS and shell type
+    ```
```

Comment:
Numbering is off in this list too.

This step looks like a repeat of the previous step, no?